components/header.js

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
+export default function Header() {
+  return (
+    <>
+      <Headerbody>
+        <AppHeadline>MediHelp</AppHeadline>
+        <AbcdeLink>ABCDE</AbcdeLink>
+      </Headerbody>
+    </>
+  );
+}
+
+const Headerbody = styled.header`
+  background-color: hotpink;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+`;
+
+const AppHeadline = styled.p``;
```

Comment:
if this becomes your headline at a later point, you should use `<h1>` instead

Reply:
*solved